pkg/pyinst/cryptography/osrandom_engine.c

```diff
@@ -0,0 +1,167 @@
+static const char *Cryptography_osrandom_engine_id = "osrandom";
```

Comment:
yes. this must be from before the pyinstaller3 release, in which a hook for gathering this was added. i'll remove it from the PR.